models/DataObject/ClassDefinition/Data/Localizedfields.php

```diff
@@ -639,15 +643,19 @@ public function save($object, $params = [])
      */
     public function load($object, $params = [])
     {
+        $theObject = $object;
```

Comment:
what is the point of this construction here?

Reply:
thx :-)